### ISSUE TITLE
Enhancement request: Option for user to set the time scale Days/Months/Years

### ISSUE THREAD

**Issue by luconsta** [microsoft/powerbi-visuals-gantt#191]
Hello,

I would like to suggest, if or when possible, a way to let the user control the time scale option from Days to Months or Years. Now I could control the value, but what if the user want another time scale?
Is there a way to let the user controll it now (maybe I miss the option somewhere)?

Kind Regards,
Lucian

**Comment by helen508light** [comment]
You can use formatting option Data type->Type. Scale would be rendered in days, months, years and etc.

**Comment by luconsta** [comment]
Hello @helen508light,

Thank you for your message, but I wasn't talking about the option for configuring the visual when editing the report. I found that option and I could set it, but it remain "fixed".

I need a way to give the user that is looking at the report to change this "on-the-fly", when the report is already published. If this is already possible, please let me know, because I could not find it. :)

Kind Regards,
Lucian

**Comment by erapade** [comment]
I totally agree this is needed. Now I have done the same using multiple gantt charts, switching between them using buttons and bookmarks. But it shall be possible to do this from within the Gantt chart itself
